### PR TITLE
Fix 2 import buttons alignment in calendar import modal

### PR DESCRIPTION
**Pull request by Jerome-Herbinet**
Signed-off-by: Jérôme Herbinet <33763786+Jerome-Herbinet@users.noreply.github.com>

**Before :** 

**After :** 

Note : I notice that the file is a .css and that it's content is written in SCSS. Should I run a "npm run sass" ?

### CODE REVIEW

**Comment by miaulalala** [comment]
> npm run sass

I think so, yes.

**Comment by Jerome-Herbinet** [comment]
> > npm run sass
> 
> I think so, yes.

@miaulalala the "npm run sass" script doesn't exist in package.json

**Comment by miaulalala** [comment]
@st3iny do you know?

**Comment by ChristophWurst** [comment]
Scss files are compiled with `npm run build`, `npm run watch` and `npm run dev`. The result is not committed.

File: css/import.scss [hunk 1]
```diff
@@ -31,6 +31,12 @@
 			text-align: center;
 		}
 
+		.import-modal__actions {
+			> button {
+				display: inline-block !important;
+			}
```
Comment (st3iny) [inline css/import.scss:37]:
```suggestion
			display: flex;
			gap: 5px;
```

Why not use flex? The button selector doesn't have to be hard coded this way.

**Comment by Jerome-Herbinet** [comment]
I've taken into consideration the @st3iny 's suggestion (simpler, so, better, thanks). 
@ChristophWurst is everything good to go ?

**Review by st3iny — APPROVED** [review]
Tested and works.

**Comment by st3iny** [comment]
stable3.5 is not affected

**Pull request merged**